app/models/review.rb

```diff
@@ -5,6 +5,10 @@ class Review < ActiveRecord::Base
   belongs_to :restaurant
   # has_many :likes, dependent: :destroy
 
+  validates :restaurant, presence: true
+
   validates :user, presence: true
 
+  validates :rating, presence: true,
+             inclusion: { in: RATINGS }
```

Comment:
What do you think of this? there's a good chance we'll be changing the RATINGS array later on.